graphene_django/tests/urls.py

```diff
@@ -1,8 +1,8 @@
-from django.conf.urls import url
+from django.urls import re_path
 
 from ..views import GraphQLView
 
 urlpatterns = [
-    url(r"^graphql/batch", GraphQLView.as_view(batch=True)),
-    url(r"^graphql", GraphQLView.as_view(graphiql=True)),
+    re_path(r"^graphql/batch", GraphQLView.as_view(batch=True)),
+    re_path(r"^graphql", GraphQLView.as_view(graphiql=True)),
```

Comment:
I think these can be just `path`.

Reply:
I'm not sure why but when I use path here it breaks test_pytest_fixture_usage